src/browser/sync/authentication_manager.ts

```diff
@@ -400,6 +410,17 @@ export class AuthenticationManager {
   }
 
   private setAuthState(newAuth: AuthState) {
+    const authStateForLog =
+      newAuth.state === "waitingForServerConfirmationOfFreshToken"
+        ? {
+            ...newAuth,
+            token: newAuth.token.slice(0, 7) + "..." + newAuth.token.slice(-7),
```

Comment:
This is meant to help clarify which token is being used. I don't believe showing the first and last 7 characters of the jwt will present a security risk, but wanted to call this out in case it does. We also may just need the last 7. This is also done in one other place in the PR for comparison.